sdk/src/wallet/events/types.rs

```diff
@@ -36,6 +36,112 @@ pub enum WalletEvent {
     TransactionProgress(TransactionProgressEvent),
 }
 
+impl Serialize for WalletEvent {
```

Comment:
Changelog and nodejs types also need to be updated?

Reply:
Done